ballerina/json_api.bal

```diff
@@ -53,10 +53,26 @@ public isolated function parseStream(stream<byte[], error?> s, Options options =
 
 # Converts a value of type `anydata` to `json`.
 #
-# + v - Source anydata value
-# + return - representation of `v` as value of type json
-public isolated function toJson(anydata v)
-        returns json|Error = @java:Method {'class: "io.ballerina.lib.data.jsondata.json.Native"} external;
+# + j - Source anydata value
+# + return - representation of `j` as value of type json
+public isolated function toJson(anydata j) returns json {
```

Comment:
Rename to `v` again, since this can break backward compatibility.

Reply:
Update via https://github.com/ballerina-platform/module-ballerina-data.jsondata/pull/35/commits/6f3c0763f7a3f35bed45cc86dd7774a5222ae02e